include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -233,7 +233,31 @@ struct layout_stride {
 
     //--------------------------------------------------------------------------------
 
-    MDSPAN_INLINE_FUNCTION_DEFAULTED constexpr mapping() noexcept = default;
+    MDSPAN_INLINE_FUNCTION_DEFAULTED constexpr mapping() noexcept
+#if defined(_MDSPAN_USE_ATTRIBUTE_NO_UNIQUE_ADDRESS)
+      : __members{
+#else
+      : __base_t(__base_t{__member_pair_t(
```

Comment:
Do we need the repeated `__base_t` here?  It's probably harmless in C++17 as the compiler can elide it, but it makes the code harder to read.

Reply:
just using the pattern from elsewhere